admin_manual/configuration/server/occ_command.rst

```diff
@@ -198,6 +198,9 @@ Disable an app
 By default all checks are enabled. 
 The Activity app is an example of a correctly-formatted app
 
+.. note::
```

Comment:
This comes in the middle of explanation an examples of ``app:check-code`` - looks a bit odd.

Reply:
I see my mistake. Thanks for pointing it out.